.env.example

```
@@ -110,6 +110,14 @@ GOOGLE_AI_API_KEY=
 LLM_PROVIDER_MODE=openrouter
 AGENT_RECURSION_LIMIT=240
 
+# Tool outputs at or below this character count enter agent context unchanged.
+TOOL_OUTPUT_PASS_THROUGH_CHARS=40000
+# Larger outputs are truncated to this character count before LLM summarization.
+# This limit is shared by the source cap and context-capture summarization paths.
+TOOL_OUTPUT_MAX_SUMMARIZATION_INPUT_CHARS=400000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TOOL_OUTPUT_MAX_SUMMARIZATION_INPUT_CHARS key should go before the TOOL_OUTPUT_PASS_THROUGH_CHARS key

(UnorderedKey)